libenkf/src/enkf_config_node.c

```diff
@@ -603,6 +607,43 @@ const char * enkf_config_node_get_enkf_infile( const enkf_config_node_type * con
   return path_fmt_get_fmt( config_node->enkf_infile_fmt );
 }
 
+
+const char * enkf_config_node_get_init_file(enkf_config_node_type * config_node, const enkf_main_type * enkf_main ) {
```

Comment:
This function should take a `model_config_type` as input - and not a full oversized `enkf_main` instance - I am honestly surprised it even compiles like this!?